VERSION.cmake

```diff
@@ -1,6 +1,6 @@
 set (DEFAULT_LIBDNF_MAJOR_VERSION 0)
 set (DEFAULT_LIBDNF_MINOR_VERSION 49)
-set (DEFAULT_LIBDNF_MICRO_VERSION 0)
+set (DEFAULT_LIBDNF_MICRO_VERSION 1)
```

Comment:
I think we should be bumping `MINOR` version when adding features and bump `MICRO` only in case of bug fixes. (please also rebase, `MINOR` version 50 is already in master)

Also, why not squash the two commits into a single one, seems too fine a granularity to split this into two commits...

Reply:
Done.